app/models/data.py

```diff
@@ -1,9 +1,3 @@
-"""This is an example model which will be used for creating stories on the web.
```

Comment:
but we wanted to keep creating stories on the web!

Reply:
https://www.contentful.com/ ;)